protocols/pcr_prep/pcr_prep.py

```diff
@@ -0,0 +1,868 @@
+                    pipette.pick_up_tip(rack_well)
+                except protocol_api.labware.OutOfTipsError:
+                    ctx.pause("Replace empty tip racks for {}".
+                              format(pipette.name))
+                    pipette.reset_tipracks()
+                for well in mastermix_destination:
+                    pipette.aspirate(volume, source)
+                    pipette.dispense(volume, well)
+                pipette.drop_tip()
+        # After the mastermix has all components added it's time to mix
+        mix_volume = mastermix_volume_sum / (2*len(mastermix_destination))
+        pipette = select_pipette(mix_volume, mastermix_destination,
+                                 mastermix_labware)
+        if is_single(pipette):
+            if pipette.has_tip:
+                pipette.drop_tips()
+            pipette.pick_up_tip()
+            for well in mastermix_destination:
+                if mix_volume > pipette.max_volume:
+                    pipette.mix(5, pipette.max_volume, well)
+                    pipette.blow_out(well)
+                else:
+                    pipette.mix(5, mix_volume, well)
+                    pipette.blow_out(well)
+        else:
+            if pipette.has_tip:
+                pipette.drop_tip()
+            rack, rack_well = \
+                get_tiprack_well_with_n_tips(pipette,
+                                             len(mastermix_destination))
+            pipette.pick_up_tip(rack_well)
+            if mix_volume > pipette.max_volume:
+                if isinstance(mastermix_destination, list):
+                    pipette.mix(5, pipette.max_volume,
+                                mastermix_destination[0])
+                    pipette.blow_out(mastermix_destination[0])
+                else:
+                    pipette.mix(5, pipette.max_volume, mastermix_destination)
+                    pipette.blow_out(mastermix_destination)
+            else:
+                if isinstance(mastermix_destination, list):
+                    pipette.mix(5, mix_volume,
+                                mastermix_destination[0])
+                    pipette.blow_out(mastermix_destination[0])
+                else:
+                    pipette.mix(5, mix_volume, mastermix_destination)
+                    pipette.blow_out(mastermix_destination)
+        ctx.comment("\nFinished mixing mastermix\n")
+
+    ctx.comment("Please insert the DNA sample plate in slot {} ({})"
+                .format(lw_slot_dict["DNA_sample_plate_step_2"],
+                        _target_labware))
+    ctx.comment("and the target plate in slot {} ({})"
+                .format(lw_slot_dict["target_plate_step_2"],
+                        _DNA_sample_plate))
+    if not _use_same_pipettes:
+        ctx.comment("Change the pipettes for step 2, left: {}, right: {}"
+                    .format(_left_pipette_part_2, _right_pipette_part_2))
+    ctx.pause()
+
+    if _create_mastermix:
```

Comment:
Not sure if this work, I should probably rewrite the protocol so that all labware and temp modules remain static throughout the run.